paths/key_links/batch_destroy.yaml

```diff
@@ -29,7 +23,10 @@ requestBody:
             example: ["child_key_id1", "child_key_id2"]
             items:
               type: string
-
+          unlink_parent:
+            description: Whether to unlink the parent key as well and unmark it as linked-key.
+            type: boolean
+            default: false
```

Comment:
Just curious around the change to `default: false` from `required: false`. The way i see it, its always sending an extra args with default value. Is it because we are using `Boolifier` in the API?

Reply:
the `default` param should be describing the API behaviour. If it's not passed the API will fallback to the default value